src/Faker/Provider/de_AT/Person.php

```diff
@@ -121,28 +123,33 @@ public static function suffix()
     /**
      * Generates a random Austrian Social Security number.
      * @link https://de.wikipedia.org/wiki/Sozialversicherungsnummer#.C3.96sterreich
+     * @param  \DateTime|null  $birthdate
      * @return string
      */
-    public static function ssn()
+    public static function ssn(\DateTime $birthdate = null)
     {
-        $birthdate = (new \DateTime('@' . mt_rand(0, time())))->format('dmy');
+        if (!$birthdate) {
```

Comment:
You may use null coalescing operator: `$birthdate = $birthdate ?? DateTime::dateTimeThisCentury();`

Reply:
Done.